storage/storage.go

```diff
@@ -175,3 +175,41 @@ type notImplemented struct {
 func (e notImplemented) Error() string {
 	return fmt.Sprintf("%q is not supported on %q storage", e.method, e.apiType)
 }
+
+type Metadata map[string]string
+
+func (m Metadata) SetACL(acl string) {
+	m["ACL"] = acl
+}
+
+func (m Metadata) ACL() string {
+	acl, ok := m["ACL"]
```

Comment:
these controls are not needed. just use
```
func (m Metadata) ACL() string {
	return m["ACL"]
}
```

Reply:
those checks are there following [this](https://github.com/peak/s5cmd/pull/185#discussion_r455538532) feedback. I thought the reasoning might be for readability. Shall I remove those checks?

Reply:
My bad, I forgot that map returns zero value for non-existent keys, you can remove them.